### PR TITLE
screen-meetings-list added

### PR DESCRIPTION
**Pull request by shahzaib3120**
## Ticket

PLAT-XXXX
_Related tickets:_
_Related PRs:_

## Type of PR

- [ ] Bugfix
- [ ] New feature
- [ ] Minor changes

## Changes introduced

_Describe the changes being introduced in this PR_
_Include screenshots if necessary_
_Please link any documentation reference that relate to the changes_

## Test and review

_Describe how a reviewer should test your PR_

### CODE REVIEW

**Comment by shahzaib3120** [comment]
preview

**Review by shahraizali — DISMISSED** [review]

File: modules/screen-meetings-list/index.js [hunk 1]
```diff
@@ -1,14 +1,254 @@
...
+    return {
+      right: index * 10
+    };
+  };
+  const giveRandomColor = () => {
+    // reutrn random grayscale color
+    const random = Math.floor(Math.random() * 255);
+
+    return {
+      // backgroundColor: "#" + Math.floor(Math.random() * 16777215).toString(16)
```
Comment (danielsousaio) [inline modules/screen-meetings-list/index.js:177]: could you remove this commented line please?

File: modules/screen-meetings-list/index.js [hunk 2]
```diff
@@ -1,14 +1,254 @@
...
-  StyleSheet
+  StyleSheet,
+  Image,
+  TextInput,
+  Pressable,
+  FlatList
 } from "react-native";
 
 const MeetingsList = () => {
+  const [seacrhText, setSearchText] = useState("");
```
Comment (danielsousaio) [inline modules/screen-meetings-list/index.js:13]:
```suggestion
  const [searchText, setSearchText] = useState("");
```

File: modules/screen-meetings-list/index.js [hunk 3]
```diff
@@ -1,14 +1,254 @@
...
+    };
+    return userStyles;
+  };
   return (
     <View style={styles.container}>
-      <Text style={styles.heading}>Meetings List</Text>
+      <Input
+        text="Search"
+        placeholder="Search"
+        value={seacrhText}
```
Comment (danielsousaio) [inline modules/screen-meetings-list/index.js:204]:
```suggestion
        value={searchText}
```

**Review by danielsousaio — CHANGES_REQUESTED** [review]

**Review by danielsousaio — APPROVED** [review]

**Pull request merged**